pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -69,13 +69,13 @@ func (o *LoginOptions) GatherInfo() error {
 // present ask for interactive user input. Will also ping the server to make sure we can
 // connect to it, and if any problem is found (e.g. certificate issues), ask the user about
 // connecting insecurely.
-func (o *LoginOptions) GatherServerInfo() error {
```

Comment:
I don't think these should be exposed since they're implementation detail.  Agree it's important to test them, but exposing them was making them seem like reusable function and I don't know that we have a use case for that now.

----- Original Message -----
> > @@ -69,13 +69,13 @@ func (o *LoginOptions) GatherInfo() error {
> >  // present ask for interactive user input. Will also ping the server to
> >  make sure we can
> >  // connect to it, and if any problem is found (e.g. certificate issues),
> >  ask the user about
> >  // connecting insecurely.
> > -func (o *LoginOptions) GatherServerInfo() error {
> 
> I agree, we don't use the full `GatherInfo` test in integration tests at the
> moment because of an issue with project caching (test assigns a project to
> user, user's project list does not reflect it right after), so we have to
> avoid `GatherProjectInfo`. There's a TODO in test_login to uncomment when
> the issue is fixed.
>  
> 
> ----- Original Message -----
> > > @@ -69,13 +69,13 @@ func (o *LoginOptions) GatherInfo() error {
> > >  // present ask for interactive user input. Will also ping the server to
> > >  make sure we can
> > >  // connect to it, and if any problem is found (e.g. certificate issues),
> > >  ask the user about
> > >  // connecting insecurely.
> > > -func (o *LoginOptions) GatherServerInfo() error {
> > 
> > Bad integration tests.  That's unit territory.
> > 
> > 
> > > On Mar 23, 2015, at 10:36 PM, Fabiano Franz <notifications@github.com>
> > > wrote:
> > > 
> > > In pkg/cmd/cli/cmd/loginoptions.go:
> > > 
> > > > @@ -69,13 +69,13 @@ func (o *LoginOptions) GatherInfo() error {
> > > >  // present ask for interactive user input. Will also ping the server
> > > >  to
> > > >  make sure we can
> > > >  // connect to it, and if any problem is found (e.g. certificate
> > > >  issues),
> > > >  ask the user about
> > > >  // connecting insecurely.
> > > > -func (o *LoginOptions) GatherServerInfo() error {
> > > Integration tests use these Gather*Info methods.
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub.
> > > 
> > 
> > 
> > ---
> > Reply to this email directly or view it on GitHub:
> > https://github.com/openshift/origin/pull/1422/files#r27002506
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1422/files#r27003159